frontend/src/components/experimentRuns/ExperimentRunsSamples.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import {connect} from "react-redux";
+
+import {Typography, List} from "antd";
+import {withSample} from "../../utils/withItem";
+import {Link} from "react-router-dom";
+import {CheckCircleTwoTone, CloseCircleTwoTone} from "@ant-design/icons";
+const {Text, Title} = Typography;
+
+import {list as listProcessMeasurements} from "../../modules/processMeasurements/actions";
+
+
+const renderSample = (sample, sampleKind) => {
+  return (
+    <>
+       <Link to={`/samples/${sample.id}`} >
+         {sample.depleted ?
+            <CloseCircleTwoTone twoToneColor="#eb2f96" /> :
+            <CheckCircleTwoTone twoToneColor="#52c41a" />
+         }
+         {' '}
+        <b>{sample.name}</b> sample ({sampleKind}){' '}
+        {sample.coordinates &&
+          `@ ${sample.coordinates}`
+        }
+      </Link>
+    </>
+  )
+}
+
+const renderProcessMeasurement = (processMeasurement) =>
+    <>{processMeasurement.volume_used} µL</>
+
+const renderListHeader = (container) => {
+  return (
+      <>
+        <Title level={5}>
+          Samples from <Link to={`/containers/${container.id}`}> {container.name} </Link>
```

Comment:
also i would replace "Samples from " by "Samples inside container " to be clearer.